src/phone_formatter.js

```diff
@@ -241,4 +329,831 @@ class PhoneFormatter extends DelimitedTextFormatter {
   }
 }
 
+const COUNTRY_CODES = [
+  {
```

Comment:
I feel that datasets like this contribute significantly towards the overall size of fieldkit. Majority of people need only a couple of values from here ever. Is it possible to pass this in and provide JSON file for reference of values to pass in?